### ISSUE TITLE
Random Forest Classification

### ISSUE THREAD

**Issue by NishkarshRaj** [NishkarshRaj/100DaysofMLCode#23]
I have programmed for **Random Forest Classification** in both R and Python.

Please help me create a explanatory markdown documentation for the same.

Link for the code is [here](https://github.com/NishkarshRaj/100DaysofMLCode/tree/master/4_Classification/Random_Forest_Classification).

**Comment by Kaif10** [comment]
I would like to solve this issue. Can you explain what kind of document do you need?

**Comment by NishkarshRaj** [comment]
@Kaif10 A basic introduction for the algorithm and brief explanation of syntax of the code.

**Comment by pragyakapoor** [comment]
Is this still open? Can help with python code documentation.

**Comment by NishkarshRaj** [comment]
@pragyakapoor Yes, its opened! We have documentation for the topic but not for the codes! It would be great if you can add an explanation for programs in Python and R or either of them

**Comment by white-peed** [comment]
@NishkarshRaj , I think the comments present in the python code is looking good. Extra comments may make the code look clumsy. Also, if we make a separate documentation, it will be quite similar to the documentation for the topic. 
So, should we create a specific documentation explaining only the python code?
